timemachine-compose-afp.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+services:
+  timemachine:
+    network_mode: "host"
+    environment:
+      - CUSTOM_AFP_CONF=false
+      - LOG_LEVEL=info
+      - MIMIC_MODEL=TimeCapsule6,106
+      - TM_USERNAME=timemachine
+      - TM_GROUPNAME=timemachine
+      - TM_UID=1000
+      - TM_GID=1000
+      - PASSWORD=timemachine
+      - SET_PERMISSIONS=false
+      - SHARE_NAME=TimeMachine
+      - VOLUME_SIZE_LIMIT=0
+    restart: unless-stopped
+    ports:
+     - "548:548"
+     - "636:636"
+    volumes:
+     - /path/to/your/timemachine/volume:/opt/timemachine
+     - ./timemachine-netatalk:/var/netatalk
```

Comment:
Same comment on the volumes here with a relative path vs a docker managed volume.

Reply:
same answer